core-bundle/src/Resources/contao/dca/tl_module.php

```diff
@@ -125,6 +125,7 @@
 		'template'                    => '{title_legend},name,headline,type;{template_legend},data,customTpl;{protected_legend:hide},protected;{expert_legend:hide},guests,cssID',
 		'rssReader'                   => '{title_legend},name,headline,type;{config_legend},rss_feed,numberOfItems,perPage,skipFirst,rss_cache;{template_legend:hide},rss_template;{protected_legend:hide},protected;{expert_legend:hide},guests,cssID',
 		'two_factor'                  => '{title_legend},name,headline,type;{template_legend:hide},customTpl;{protected_legend:hide},protected;{expert_legend:hide},cssID',
+		'root_page_dependent_module'  => '{title_legend},name,type;{config_legend},rootPageDependentModules;{template_legend:hide},customTpl;{protected_legend:hide},protected;{expert_legend:hide},cssID'
```

Comment:
```suggestion
		'root_page_dependent_module'  => '{title_legend},name,type;{config_legend},rootPageDependentModules;{template_legend:hide},customTpl;{protected_legend:hide},protected'
```

The CSS classes and CSS ID are never output since the template does not extend from `block_searchable` or `block_unsearchable` - or will be removed altogether (https://github.com/contao/contao/pull/3613/files#r777346193).

Reply:
We could extend from `block_unsearchable` though (see https://github.com/contao/contao/pull/3613#discussion_r777360303).

Reply:
Removed in e97704c2f, as the rendered module itself brings its own classes (see https://github.com/contao/contao/commit/86cbde12746d431da23cabc1b8dff82d41bd0440 as well).